docs/development/releasing.rst

```diff
@@ -115,7 +115,12 @@ packages that use the full bugfix/maintenance branch approach.)
 
    Don't forget to remove any non-committed files with::
 
-      $  git clean -dfx
+      $ git clean -dfx
+
+#. Make sure the source distribution doesn't inherit limited permissions
+   following you default umask::
+
+     $ umask 0022
```

Comment:
@bsipocz  I don't think changing the `umask` here does anything if the local files have already been downloaded/cloned with the limited `umask` permissions.  I think the `umask` needs to be changed before cloning the repo.

When I released `photutils` it was the local file permissions need to be changed/checked before running `python setup.py build sdist`.  In other words, the files in the source tarball inherited the same permissions of the local files independent of my `umask` setting.  The issue was that the local files got their limited permissions from my limited `umask` when I first cloned the repo.